packages/firmware/src/gps.cpp

```diff
@@ -0,0 +1,12 @@
+#include "gps.h"
+
+gps_loc_t getGPS(Adafruit_FONA* fona) {
+    gps_loc_t location;
+    location.success = fona->getGPS(&location.latitude,
+                                    &location.longitude,
+                                    &location.speed_kph,
+                                    &location.heading,
+                                    &location.altitude);
+    location.speed_mph = location.speed_kph * 0.621371192;
```

Comment:
How do we feel about just storing speed in kph instead of converting it? Mostly just to use SI units rather than some other benefit.

Reply:
I'm fine with that, that is computation that is probably better off done server side anyway. This PR is largely just me implementing some examples and spitballing some things. It will likely change drastically once I start using the real devices.